doc/reduce.md

```diff
@@ -12,30 +12,104 @@ grppi::reduce(exec, other_arguments...);
 
 There is a single variant of the reduction:
 
-* Sequence reduction: Takes a sequence represented by two iterators and reduces the values.
+* **Sequence reduction without initial value**: Reduces a sequence of values
```

Comment:
I think that this is not true anymore. Now, this interface allways requires the identity value.  This applies also for its description.